### ISSUE TITLE
/Resources/Private/Partials/[Frontend plugin]/Properties.html isn't correctly generated

### ISSUE THREAD

**Issue by zeichensatz** [FriendsOfTYPO3/extension_builder#237]
After saving an extension, I realized, that the Show-action for some frontend plugins didn't display the values, only the field names.
The corresponding /Resources/Private/Partials/[Frontend plugin]/Properties.html should read like the following example:
```
<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
<table class="tx-wok-test" >
        <tr>
                <td>
                        <f:translate key="tx_woktest_domain_model_test1.name" />
                </td>
                <td>
                {test1.name}
                </td>
        </tr>
</table>
</html>
```
I figured out, that the {test1.name} is missing occassionally after saving the extension and it then reads like this:
```
<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
<table class="tx-wok-test" >
        <tr>
                <td>
                        <f:translate key="tx_woktest_domain_model_test1.name" />
                </td>
                <td>
                </td>
        </tr>
</table>
</html>
```
When I delete all caches before saving the extension, at least the code for the first defined plugin is generated properly.
If more than one plugin is defined, all others miss the values too.

[TYPO3 9.5.11]
[Extension builder 9.10.2]

**Comment by nicodh** [comment]
Yes that's a caching issue. If these templates are "prerendered" and parsed the marker get lost. We have to find a workaround for that...

**Comment by zeichensatz** [comment]
As I realized now, the problem only seems to occur for the generation of the frontend plugins. The generation of the backend modules (the files Properties.html in Resources/Private/Backend/... seems to work without missing the marker. Perhaps this helps?

**Comment by ohader** [comment]
Addressed in #242 commit 72b056b449e2accf9818bf4b1e6e266fa2ddb413

**Comment by ohader** [comment]
Merged #242
